app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,31 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/wallpaper_main_screen"
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/title_text"
+        style="@style/TitleStyleMainActivity"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/main_activity_text_title"
+        android:layout_marginStart="@dimen/text_main_activity_margin_start"
+        android:layout_marginEnd="@dimen/text_main_activity_margin_end"
+        android:layout_marginTop="@dimen/text_main_activity_margin_top"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"/>
+
+    <Button
+        android:id="@+id/set_button"
+        style="@style/ButtonStyleMainActivity"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
+        android:text="@string/main_activity_button_set_size"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/title_text"/>
 
 </androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
add an empty line at the end of the file to avoid this error

Reply:
@mariano98montero you can set Android Studio to to this automatically

settings -> search for empty line

Reply:
done, and applied the setting to my IDE, thanks!